graylog2-server/src/main/java/org/graylog2/commands/Server.java

```diff
@@ -177,7 +177,7 @@ protected List<Module> getCommandBindings(FeatureFlags featureFlags) {
                 new RetentionStrategyBindings(elasticsearchConfiguration),
                 new PeriodicalBindings(),
                 new ObjectMapperModule(chainingClassLoader),
-                new RestApiBindings(configuration),
+                new RestApiBindings(configuration, featureFlags.isOn("field_types_management")),
```

Comment:
I expect that the whole feature flag will be gone. It is temporary, so that if we merge that stuff no client will have this feature prematurely. So I'd not focus too much on discussing how to pass this temporary flag.